mne/beamformer/tests/test_lcmv.py

```diff
@@ -666,46 +667,55 @@ def test_localization_bias_fixed(bias_params_fixed, reg, weight_norm, use_cov,
+        (0.05, 'vector', 'nai', True, None, 35, 37, 0.98, 0.99),
+        (0.05, 'vector', None, True, None, 12, 14, 0.97, 0.98),
+        (0.05, 'vector', None, True, 0.8, 39, 43, 0.97, 0.98),
+        (0.05, 'max-power', 'unit-noise-gain-invariant', False, None, 17, 20, 0, 0),  # noqa: E501
+        (0.05, 'max-power', 'unit-noise-gain', False, None, 17, 20, 0, 0),
+        (0.05, 'max-power', 'nai', True, None, 21, 24, 0, 0),
+        (0.05, 'max-power', None, True, None, 7, 10, 0, 0),
+        (0.05, 'max-power', None, True, 0.8, 15, 18, 0, 0),
+        (0.05, None, None, True, 0.8, 40, 42, 0, 0),
         # no reg
-        (0.00, 'vector', None, True, None, 21, 32),
-        (0.00, 'vector', 'unit-noise-gain-invariant', True, None, 50, 65),
-        (0.00, 'vector', 'unit-noise-gain', True, None, 42, 65),
-        (0.00, 'vector', 'nai', True, None, 42, 65),
-        (0.00, 'max-power', None, True, None, 13, 19),
-        (0.00, 'max-power', 'unit-noise-gain-invariant', True, None, 43, 50),
-        (0.00, 'max-power', 'unit-noise-gain', True, None, 43, 50),
-        (0.00, 'max-power', 'nai', True, None, 43, 50),
+        (0.00, 'vector', None, True, None, 21, 32, 0.93, 0.94),
+        (0.00, 'vector', 'unit-noise-gain-invariant', True, None, 50, 65, 0.90, 0.92),  # noqa: E501
+        (0.00, 'vector', 'unit-noise-gain', True, None, 42, 65, 0.96, 0.98),
+        (0.00, 'vector', 'nai', True, None, 42, 65, 0.96, 0.98),
+        (0.00, 'max-power', None, True, None, 13, 19, 0, 0),
+        (0.00, 'max-power', 'unit-noise-gain-invariant', True, None, 43, 50, 0, 0),  # noqa: E501
+        (0.00, 'max-power', 'unit-noise-gain', True, None, 43, 50, 0, 0),
+        (0.00, 'max-power', 'nai', True, None, 43, 50, 0, 0),
     ])
 def test_localization_bias_free(bias_params_free, reg, pick_ori, weight_norm,
-                                use_cov, depth, lower, upper):
+                                use_cov, depth, lower, upper, lo, uo):
     """Test localization bias for free-orientation LCMV."""
     evoked, fwd, noise_cov, data_cov, want = bias_params_free
     if not use_cov:
         evoked.pick_types(meg='grad')
         noise_cov = None
-    loc = apply_lcmv(evoked, make_lcmv(evoked.info, fwd, data_cov, reg,
-                                       noise_cov, pick_ori=pick_ori,
-                                       weight_norm=weight_norm,
-                                       depth=depth)).data
+    filters = make_lcmv(evoked.info, fwd, data_cov, reg,
+                        noise_cov, pick_ori=pick_ori,
+                        weight_norm=weight_norm,
+                        depth=depth)
+    loc = apply_lcmv(evoked, filters).data
+    if pick_ori == 'vector':
+        ori = loc.copy() / np.linalg.norm(loc, axis=1, keepdims=True)
+    else:
+        # doesn't make sense for pooled (None) or max-power (can't be all 3)
+        ori = None
     loc = np.linalg.norm(loc, axis=1) if pick_ori == 'vector' else np.abs(loc)
```

Comment:
Wait, now I am confused. Didn't we set out to test whether the chosen orientation matches the simulated orientation? Does this just test the orientation the leadfield is giving?

Reply:
The "simulated data" here is just the leadfield, and the (free ori) leadfield is oriented as X0, Y0, Z0, X1, Y1, Z1, ... , XN, YN, ZN for the N+1 sources, so the correct orientation of the simulated data is just `np.eye(3)` repeating over and over again

Reply:
But we are not testing the actual selected orientation in the scalar beamformer cases, or are we?

Reply:
The `max_power_ori` can't possibly work in this situation -- it selects a single orientation for each location, and we are simulating activity in each of the three cardinal directions. We probably should have a separate test for example that if we simulate data in the normal direction, the max_power_ori beamformer recovers the correct orientations. I'll see if it's easy to add, should be...

Reply:
(can't possibly work to get the orientations right anyway)

Reply:
Okay, I see. If we want to check for `real` in the orientation selection for the DICS beamformers, then we would indeed need a test with a single orientation.